pkg/api/v20191231preview/openshiftcluster_validatestatic.go

```diff
@@ -126,6 +127,25 @@ func (sv *openShiftClusterStaticValidator) validateProperties(path string, p *Pr
 	return nil
 }
 
+func (sv *openShiftClusterStaticValidator) validateClusterProfile(path string, cp *ClusterProfile) error {
+	if !rxDomainName.MatchString(cp.Domain) {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, path+".domain", "The provided domain '%s' is invalid.", cp.Domain)
+	}
+	switch cp.Version {
+	case "", "4.3.0-0.nightly-2019-12-05-001549":
```

Comment:
I expect this to change in the future (?)

Reply:
yes